libdnf/base/transaction.cpp

```diff
@@ -145,8 +144,7 @@ void Transaction::Impl::add_resolve_log(
 void Transaction::Impl::add_resolve_log(
     GoalProblem problem, std::vector<std::vector<std::pair<libdnf::ProblemRules, std::vector<std::string>>>> problems) {
     resolve_logs.emplace_back(LogEvent(problem, problems));
-    // TODO(jmracek) Use a logger properly and change a way how to report to terminal
-    std::cout << resolve_logs.back().to_string() << std::endl;
+    // TODO(jmracek) Use a logger properly
```

Comment:
FTR I've been annoyed by these before, but I wasn't sure they can be removed without a proper replacement. (aren't they required in microdnf output?)

@j-mracek

Reply:
There is one thing still missing (I'll fix it in this PR) - resolve logs are not presented to the user in case the transaction is correctly resolved (ends with `libdnf::GoalProblem::NO_PROBLEM`). In case of problems they are printed via `GoalResolveError` exception.

Reply:
Fixed. Now the resolve logs are printed together with transaction table.